src/LogViewer/index.js

```diff
@@ -149,7 +149,8 @@ export default class LogViewer extends React.Component {
 
   renderChunk(height, index, shouldLoad) {
     const load = shouldLoad(index);
-    const { worker } = this.state;
+    const { worker, highlightStart, highlightEnd } = this.state;
+    const metadata = { highlightStart, highlightEnd };
```

Comment:
We can probably lose this variable creation since it's only used in the `postMessage` call. I'll note that there.